Core/Utilities/Graphs/GraphData.py

```diff
@@ -401,7 +401,7 @@ def initialize( self ):
       self.keys = self.sortKeys()
 
     self.values = [ self.parsed_data[k] for k in self.keys ]
```

Comment:
ops sorry: self.values = [ self.parsed_data[k] if k in self.parsed_data else 0.0 for k in self.keys ]